docs/user_guide/configuring.rst

```diff
@@ -150,3 +150,22 @@ use the following configuration:
    html_theme_options = {
      "navigation_with_keys": False
    }
+
+
+Show more levels of the in-page TOC by default
+==============================================
+
+Normally only the 2nd-level headers of a page are show in the right
+table of contents, and deeper levels are only shown when they are part
+of an active section (when it is scrolled on screen).
+
+You can show deeper levels by default by using the following configuration:
+
+.. code-block:: python
+
+   html_theme_options = {
+     "show_toc_level": 2
```

Comment:
The "2" here might be a bit confusing with the above "only the 2nd-level headers ..." ?

Reply:
Yeah I wasn't quite sure what was the most intuitive thing to use here. The "2" could either refer to the level of the header in the document, OR it could refer to the level of the in-page TOC to the right. AKA the confusion is because the 2nd-level of the in-page TOC refers to the *third* level headers on the page. Do you have an intuition for which would be less-confusing for people?

Reply:
I think as config value, "2" is certainly fine here, in the meaning: "show 2 levels in the toc", which I think is the clearest definition (you don't need to know how to start counting the levels, include the main title as 1st level or not?)

Reply:
Actually, if there are multiple 1st level headers, it are actually those that are shown in the TOC. It's only the 2nd level headers that are shown if there is a single main title. So maybe it's better to speak about "only the 1st level headers (after the main title) of a page are shown"?

Reply:
I liked the language that you added, to make it clear that the configuration is for how many TOC levels are *displayed* (whether they begin with first-level headers or second-level headers, it'll remain consistent then)